src/appleseed/renderer/modeling/object/sphereobject.h

```diff
@@ -0,0 +1,135 @@
+
+//
+// This source file is part of appleseed.
+// Visit https://appleseedhq.net/ for additional information and resources.
+//
+// This software is released under the MIT license.
+//
+// Copyright (c) Mayank Dhiman, The appleseedhq Organization
```

Comment:
Year missing. Please follow the conventions outlined in https://github.com/appleseedhq/appleseed/wiki/Copyright-Notices and used in all other source files.